i18n/Aliases/Aliases.php

```diff
@@ -1,12 +1,15 @@
 <?php
+
 $specialPageAliases = [];
 
+/** English (English) */
 $specialPageAliases['en'] = [
 	'ManageApprovers' => [ 'ManageApprovers' ],
 	'PendingApprovals' => [ 'PendingApprovals' ],
 ];
 
-$specialPageAliases['qqq'] = [
-	'ManageApprovers' => [ 'ManageApprovers' ],
-	'PendingApprovals' => [ 'PendingApprovals' ],
+/** German (Deutsch) */
+$specialPageAliases['de'] = [
+	'ManageApprovers' => [ 'Genehmiger_verwalten' ],
+	'PendingApprovals' => [ 'Ausstehende_Genehmigungen' ],
```

Comment:
Underscores? Is that in-line with other German special pages?!

Reply:
Yes, e.g. MW https://github.com/wikimedia/mediawiki/blob/master/languages/messages/MessagesDe.php#L38 and SMW https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/i18n/extra/SemanticMediaWiki.alias.php#L145